FE/src/hooks/index.ts

```diff
@@ -6,3 +6,9 @@ export const getFormalizedDate = (dateString: string) => {
 
   return `${year}.${month}.${date}`;
 };
+
+export const getText = (html: string) => {
```

Comment:
함수명을 이렇게 바꾸면 함수의 역할을 알기 더 좋을 거 같아요.

```suggestion
export const getTextContentFromHTML = (html: string) => {
```